src/Distributions.jl

```diff
@@ -2,33 +2,36 @@ __precompile__(true)
 
 module Distributions
 
-using PDMats
-using StatsFuns
-using StatsBase
-using Compat
-import Compat.MathConstants: γ
-
-import QuadGK.quadgk
-import Compat.view
-import Base.Random
-import Base: size, eltype, length, full, convert, show, getindex, scale!, rand, rand!
-import Base: sum, mean, median, maximum, minimum, quantile, std, var, cov, cor
-import Base: +, -
-import Base.Math.@horner
-import Base.LinAlg: Cholesky
-import Base.Random: GLOBAL_RNG, RangeGenerator, RangeGeneratorInt
-
-if isdefined(Base, :scale)
-    import Base: scale
+using StatsBase, PDMats, StatsFuns
+
+import QuadGK: quadgk
+import Base: size, eltype, length, full, convert, show, getindex, rand
+import Base: sum, maximum, minimum, +, -
+import Base.Math: @horner
+
+using LinearAlgebra, Printf
+
+using Random
+import Random: GLOBAL_RNG, RangeGenerator, rand!, SamplerRangeInt
+
+if VERSION >= v"0.7.0-beta.85"
+    import Statistics: mean, median, quantile, std, var, cov, cor
+elseif VERSION >= v"0.7.0-DEV.5238"
+    import StatsBase: mean, median, quantile, std, var, cov, cor
+else
+    import Base: mean, median, quantile, std, var, cov, cor    
```

Comment:
Not a big deal since CI doesn't fail on it here but there's some trailing whitespace here